jq.yaml

```diff
@@ -79,3 +85,7 @@ test:
       runs: |
         echo '[{"id":1,"name":"John"},{"id":2,"name":"Jane"}]' | jq '{users: map({userId: .id, userName: .name})}' | grep -E '"userId": 1|"userId": 2' || exit 1
     - uses: test/tw/ldd-check
+    - name: CVE-2024-53427
+      runs: |
+        output=$(echo '["NaN1"]' | jq -r '.[] | try (fromjson | isnan) catch .')
+        [ "$output" = "Invalid numeric literal at EOF at line 1, column 4 (while parsing 'NaN1')" ]
```

Comment:
It might be useful to have a positive test here for the "Nan" case, just to make sure we haven't broken NaN parsing completely with the cherry-pick.

Reply:
agreed, let me add that too in a follow up.

Reply:
https://github.com/wolfi-dev/os/pull/47589